nitter.nimble

```diff
@@ -14,15 +14,15 @@ requires "nim >= 1.4.8"
 requires "jester#baca3f"
 requires "karax#5cf360c"
 requires "sass#7dfdd03"
-requires "nimcrypto#4014ef9"
+requires "nimcrypto#a079df9"
```

Comment:
this bumps your `requires nim` to [`requires "nim >= 1.6"`](https://github.com/cheatfate/nimcrypto/blob/a079df92424968d46a6ac258299ce9380aa153f2/nimcrypto.nimble#L11), nitter doesn't build anymore with nim 1.4.8 because of this.

Reply:
Nice catch, thanks. 1.6.0 was released two full years ago, so I think it's time to bump that anyway.

Reply:
I have created a PR for that: https://github.com/zedeus/nitter/pull/1053